conjureup/controllers/newcloud/common.py

```diff
@@ -87,51 +87,57 @@ def save_creds(cloud, credentials):
 def is_lxd_ready():
```

Comment:
This no longer actually returns anything, nor does anything but call `lxd_init`.  Why not just inline that?  The "is ready" name is misleading.